backend/data/src/main/kotlin/io/tolgee/model/LanguageStats.kt

```diff
@@ -35,6 +36,10 @@ class LanguageStats(
   override var translatedPercentage: Double = 0.0
 
   override var reviewedPercentage: Double = 0.0
+
+  override val lastModified: Date
+    get() = updatedAt ?: Date()
+
```

Comment:
`updatedAt` is nullable in the parent class. We have the option of force unwrapping the value or we make the field nullable in all other DTOs or classes.